pyscal/pyscalcli.py

```diff
@@ -149,6 +150,31 @@ def get_parser() -> argparse.ArgumentParser:
             "Implicit for gas-water input."
         ),
     )
+
+    # Plotting arguments
+    parser.add_argument(
+        "--plot",
+        action="store_true",
+        default=False,
+        help=("Make and save relative permeability figures."),
+    )
+    parser.add_argument(
+        "--pc",
+        action="store_true",
+        default=False,
+        help=("Make and save capillary pressure figures."),
+    )
+    parser.add_argument(
+        "--semilog",
+        action="store_true",
+        default=False,
+        help=(
+            "Plot relative permeability figures with log y-axis."
+            "Run both with and without this flag to plot"
+            "both linear and semi-log relperm plots."
+        ),
+    )
```

Comment:
Similar thing, should we have a more explicit name?

Reply:
Also done in  a42fc88